applications/longshot/main.tf

```diff
@@ -44,30 +44,11 @@ module "app" {
   with_newrelic          = false
 }
 
-module "database" {
-  source = "../../components/mariadb_instance"
-
-  deprecated = var.deprecated
-
-  name              = var.name
-  environment       = var.environment
-  database_name     = "longshot"
-  instance_class    = var.environment == "production" ? "db.t2.medium" : "db.t2.micro"
-  allocated_storage = var.environment == "production" ? 100 : 5
-
-  subnet_group    = "default-vpc-7899331d"
-  security_groups = ["sg-c9a37db2"]
-}
```

Comment:
Note that deleting this resource will [automatically take a final snapshot](https://github.com/DoSomething/infrastructure/blob/fe48a2ebe17fff8630ded7b5f151c7191434250c/components/mariadb_instance/main.tf#L136).